automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -169,21 +158,19 @@ object Leonardo extends RestClient with LazyLogging {
     }
   }
 
-
   sealed trait ApiVersion {
-    override def toString: String
     def toUrlSegment: String
   }
 
   object ApiVersion {
```

Comment:
Can this be removed?

Reply:
I actually prefer to keep this unless we no longer do `v*` in our apis anymore(say we have v3 etc in the future)...

Reply:
I decided to keep `ApiVersion` because we do have versioned endpoints -- `createCluster` is v2 and everything else is v1.